pkg/pagination/graphql/models.go

```diff
@@ -0,0 +1,14 @@
+// Pager represents params from list request
+type Pager struct {
```

Comment:
I dont think this will be in GraphQL, this is the db layer abstraction to simplify the logic there